### PR TITLE
[JFG-852] Moved definition of DatabaseServiceImpl bean to a centralised place

### PR DESCRIPTION
**Pull request by a-badaev**

### CODE REVIEW

File: chassis/configuration/configuration/master/invoker/example-external-resources-invoker.conf.xml [hunk 1]
```diff
@@ -131,6 +131,7 @@
 
     <bean id="googlePageNew" class="com.griddynamics.jagger.invoker.http.HttpInvokerConfiguration">
         <property name="query" ref="googlePageQuery"/>
+        <property name="endpoint" value="http://www.google.com/search"/>
     </bean>
```
Comment (dlatnikov) [inline chassis/configuration/configuration/master/invoker/example-external-resources-invoker.conf.xml]: Is it related to the database service?

File: chassis/core/src/main/java/com/griddynamics/jagger/monitoring/MonitoringTask.java [hunk 2]
```diff
@@ -44,7 +43,6 @@ public String getParentTaskId() {
     }
 
     @Override
-    @Required
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/monitoring/MonitoringTask.java]: Why is it not required any more?

Comment (a-badaev) [inline chassis/core/src/main/java/com/griddynamics/jagger/monitoring/MonitoringTask.java]: That is actually a bug which poped-up after I added "<context:annotation-config/>" (this line in Spring context activates Spring annotations like this one.). This property is declared as required but is never set in Spring context in all declarations. So, I just removed it.

File: chassis/configuration/configuration/master/invoker/example-external-resources-invoker.conf.xml (hunk 1 above)
Comment (a-badaev) [inline chassis/configuration/configuration/master/invoker/example-external-resources-invoker.conf.xml]: This poped-up after I added "context:annotation-config"  - which activates Spring annotations like Required. So, in code it is marked as required but here it is not set as supposed to.

**Comment by dlatnikov** [comment]
+1

**Comment by amikryukov** [comment]
+1.

**Pull request merged**